easyjson.go

```diff
@@ -0,0 +1,39 @@
+package elk
+
+import (
+	"entgo.io/ent/entc/gen"
+	"github.com/mailru/easyjson/bootstrap"
+	"path/filepath"
+)
+
+func GenerateEasyJSON(next gen.Generator) gen.Generator {
+	return gen.GenerateFunc(func(g *gen.Graph) error {
+		// Let ent create all the files.
+		if err := next.Generate(g); err != nil {
+			return err
+		}
+
+		// We want to render every response struct created with easyjson.
+		ns := make([]string, 0)
```

Comment:
```suggestion
		var ns []string
```

Or use make with `len(g.Nodes) * 4`

Reply:
Thanks!